[Bug] Massdrop CTRL Endgame - Build Error 'undefined reference to g_suspend_state'

## Describe the Bug
Massdrop CTRL Endgame: Firmware building fails with the error 'undefined reference to g_suspend_state'

It can be resolved by deleting the reference to g_suspend_state from keymap.c and keymap.h in the Endgame files

It looks like it might be related to this (May 21 breaking changes): [https://github.com/qmk/qmk_firmware/commit/1646c0f26cfa21a7023d404008e4d0aa4917193d ](https://github.com/qmk/qmk_firmware/commit/1646c0f26cfa21a7023d404008e4d0aa4917193d )
One of the changes mentions USB Suspend changes, which points to this pull request: [https://github.com/qmk/qmk_firmware/pull/12697/files](https://github.com/qmk/qmk_firmware/pull/12697/files)

## System Information
 - Keyboard:  
   - Massdrop CTRL

## Additional Context
The error was confirmed by two other users helping me:
![1unknown](https://user-images.githubusercontent.com/33355340/127614335-7201fb93-4c58-4f52-8ef3-3080ef2f3cde.png)
![2image0](https://user-images.githubusercontent.com/33355340/127614340-4bac2c9e-4914-48dd-8934-6a1d4124bd26.jpg)

--- a/keyboards/massdrop/ctrl/keymaps/endgame/keymap.c
+++ b/keyboards/massdrop/ctrl/keymaps/endgame/keymap.c
@@ -373,7 +373,7 @@ void set_layer_color(int layer) {
 }
 
 void rgb_matrix_indicators_user(void) {
-    if (g_suspend_state || disable_layer_color ||
+    if (disable_layer_color ||
         rgb_matrix_get_flags() == LED_FLAG_NONE ||
         rgb_matrix_get_flags() == LED_FLAG_UNDERGLOW) {
             return;

--- a/keyboards/massdrop/ctrl/keymaps/endgame/keymap.h
+++ b/keyboards/massdrop/ctrl/keymaps/endgame/keymap.h
@@ -33,7 +33,6 @@
 #define RGB_TIME_OUT_MIN 10
 #define RGB_TIME_OUT_STEP 10
 
-extern bool g_suspend_state;
 extern rgb_config_t rgb_matrix_config;
 bool disable_layer_color;
 

